### DISCUSSION TITLE
DialogService.ShowFileDisplayDialog first display issue

### DISCUSSION THREAD

**Discussion by cmt-yamazaki** [fgilde/MudBlazor.Extensions#41] [General]
Hi.

When displaying a PDF file with ShowFileDisplayDialog,
An error message will be displayed for a moment, and then the display will be reduced.

This problem only occurs the first time when starting Blazor Server.

![image](https://github.com/fgilde/MudBlazor.Extensions/assets/32287955/cbceaea0-20d9-4779-b6af-c0ced1de09d7)

![image](https://github.com/fgilde/MudBlazor.Extensions/assets/32287955/1390ba83-fbd8-4bed-b6e0-0bc7ab66f9fb)

**Comment by fgilde** [comment]
please try to include styles manually by adding this line to you _host.cshtml
`<link id="mudex-styles" href="_content/MudBlazor.Extensions/mudBlazorExtensions.min.css" rel="stylesheet">`

If you have done it you should setup MudEx with folliwing config

`builder.Services.AddMudServicesWithExtensions(c => c.WithoutAutomaticCssLoading());`

**Comment by cmt-yamazaki** [comment]
thank you.
Improved by manually including the styles.
